requests-for-comments/2022-11-02-onboarding-2.0-continued.md

```diff
@@ -0,0 +1,121 @@
+  - The onboarding flow doesn't provide value after completion, while it could be kept as reference to specific tutorials or docs in the manual
+  - The bookmarklet is currently used to skip the onboarding and skews the metrics
+  - The onboarding is pretty boring, we could do with more hedgehogs
+- Introducing users to other products:
+  - Onboarding flow does not take into account session recording or feature flags, but only event tracking
+  - We currently have an activation checklist prompt, but it has performed poorly, and we might want to integrate this directly in the onboarding flow as additional non-blocking steps we suggest users to perform
+  - We have developed a product tour but it hasn't been released to production yet, it could be added as a final optional step after the onboarding
+
+## Success criteria
+*How do we know if this is successful (i.e. metrics, customer feedback), what's out of scope, whats makes this ambitious?*
+
+### Metrics
+- Improve the organization signup to discovery activation metric by 100% as compared to the current baseline ([see here, internal only](https://app.posthog.com/dashboard/54139))
+- Improve the organization session recordings activation metric by 50% as compared to current baseline (see [this insight, internal only](https://app.posthog.com/insights/YdcM8sBK), from Step 2 to Step 3)
+
+### Desired state
+- New users are immediately vetted for technical ability (to install the snippet) and, if they are not technical, they invite someone who is
+- Email sequences get the right people into the product to support successful onboarding
+- Customers are introduced to other products in a secondary product onboarding (eg. Sentry's QuickStart panel)
+- Onboarding flow is not visually boring and has more hedgehogs
+
+#### New onboarding flow
+- Signup flow is seamless, only email field and SSO are shown to users, any other data capturing happens within the onboarding flow
+- At the beginning of the onboarding flow, we should ask the users if they need technical help, and suggest to invite a team member
+- At any step during the onboarding, the CTA to invite members should be prominent, to avoid users getting stuck
+- The bookmarklet should be removed, and we could add an option to create some basic test events
+- Event ingestion should be the first step of a multi-step onboarding flow, as well as the only mandatory one
+- After event ingestion is completed, we should show an activation checklist button within the sidebar, to prompt the user to complete optional onboarding steps
+- Setting up session recordings should be the first optional step after event ingestion, with feature flags to be added in a second iteration
+- The current activation checklist steps should be refactored in in this new onboarding (invite users to track their marketing websites, invite users to track custom actions, to install apps, and to invite other team members)
```

Comment:
```suggestion
- The current activation checklist steps should be refactored in this new onboarding (invite users to track their marketing websites, invite users to track custom actions, to install apps, and to invite other team members)
```

Reply:
Can you clarify what you mean here? The checklist is staying within the product, correct?

Reply:
I developed the activation checklist as a series of prompts, but they don't seem to perform that well.

Check the Sentry screenshots, I'd rather have those suggestions (add tracking to your marketing website, etc.) as optional onboarding steps we can show in a sidebar, similar to what Sentry does.